c7n/filters/related.py

```diff
@@ -79,6 +80,11 @@ def process_resource(self, resource, related):
                     resource[model.id],
                     self.RelatedResource.rsplit('.', 1)[-1],
                     rid)
+                # in the event that the filter is looking specifically for absent values, we can
```

Comment:
previously, you would have to do a not present filter to get this same type of functionality as the related resource's attributes would never be evaluated, e.g.

```yaml
policies:
  - name: foo
    resource: aws.foo
    filters:
      - not:
        - type: a-related-filter
          key: foo
          value: present
```

versus

```yaml
policies:
  - name: foo
    resource: aws.foo
    filters:
        - type: a-related-filter
          key: foo
          value: absent
```